src/odin/maneuversbuilder.cc

```diff
@@ -55,6 +56,49 @@ void CountAndSortExitSignList(std::vector<Sign>* prev_signs, std::vector<Sign>*
   SortExitSignList(curr_signs);
 }
 
+uint16_t GetExpectedTurnLaneDirection(Maneuver& maneuver) {
```

Comment:
determines the expected turn lane direction based on the specified maneuver type